reserver/reserver_obj.py

```diff
@@ -101,15 +102,22 @@ def upload(self, package_name, user_parameters=None):
                     check_output(command, shell=True)
             except CalledProcessError as e:
                 publish_failed = True
-                error = e.__str__()
+                error = e.output
+                try:
+                    error = error.decode(chardet.detect(error)['encoding'])
+                except:
+                    error = error.decode('utf-8')
                 if command == commands[-2]:
                     if "403" in error and "Invalid or non-existent authentication information" in error:
                         error = "Invalid or non-existent authentication information(PyPI API Key)."
                     if "400" in error and "too similar to an existing project" in error:
                         error = "Given package name is too similar to an existing project in PyPI."
+                    if "400" in error and "isn't allowed." in error:
+                        error = "Given package name has conflict with the module name of a previously taken package (the taken package itself has a different name, but it's module name has conflict)."
```

Comment:
This error message is long and a bit vague.

My suggestion:

<p>Given package name has conflict with the module name of a previously taken package.<s>(the taken package itself has a different name, but it's module name has conflict)</s></p>